2024-10-18T12:47:06.172Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1011ms
2024-10-18T12:48:31.703Z [[31merror[39m]: Error al conectar a la base de datos: (conn:302558, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1004ms
2024-10-18T12:49:50.132Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1014ms
2024-10-18T13:02:22.106Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1012ms
2024-10-18T13:07:10.368Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1007ms
2024-10-18T13:07:32.756Z [[31merror[39m]: Error al conectar a la base de datos: (conn:302918, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1097ms
2024-10-18T13:13:27.700Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1005ms
2024-10-18T13:13:55.929Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1016ms
2024-10-18T13:14:11.900Z [[31merror[39m]: Error al conectar a la base de datos: (conn:303962, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1014ms
2024-10-18T13:23:26.979Z [[31merror[39m]: Error al conectar a la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1002ms
2024-10-18T13:23:40.270Z [[31merror[39m]: Error al conectar a la base de datos: (conn:304980, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1012ms
2024-10-18T14:02:57.728Z [[31merror[39m]: Error al conectar la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1004ms
2024-10-18T14:03:19.123Z [[31merror[39m]: Error al conectar la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1003ms
2024-10-18T14:03:19.126Z [[31merror[39m]: Error durante la inicializaci√≥n del servidor: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1003ms
2024-10-18T14:03:19.126Z [[31merror[39m]: Express no ha sido inicializado correctamente.
2024-10-18T14:03:59.683Z [[31merror[39m]: Error al conectar la base de datos: (conn:306300, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1011ms
2024-10-21T00:55:59.506Z [[31merror[39m]: Error al conectar la base de datos: (conn:400359, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1001ms
2024-10-22T21:14:39.587Z [[31merror[39m]: Error obteniendo registro por ID: Registro no encontrado
2024-10-22T21:14:39.588Z [[31merror[39m]: Error eliminando registro: Registro no encontrado
2024-10-22T21:14:39.597Z [[31merror[39m]: Error eliminando empresa: Registro no encontrado
2024-10-22T21:29:15.126Z [[31merror[39m]: Error obteniendo registros: (conn:480146, no: 1146, SQLState: 42S02) Table 'megasys2.Empresas' doesn't exist
sql: SELECT `id_empresa`, `nombre`, `createdAt`, `updatedAt` FROM `Empresas` AS `Empresa` ORDER BY `Empresa`.`nombre` ASC; - parameters:[]
2024-10-22T21:29:15.127Z [[31merror[39m]: Error listando empresas: (conn:480146, no: 1146, SQLState: 42S02) Table 'megasys2.Empresas' doesn't exist
sql: SELECT `id_empresa`, `nombre`, `createdAt`, `updatedAt` FROM `Empresas` AS `Empresa` ORDER BY `Empresa`.`nombre` ASC; - parameters:[]
2024-10-23T10:34:26.732Z [[31merror[39m]: Error en la autenticaci√≥n:
2024-10-23T10:34:43.076Z [[31merror[39m]: Error en la autenticaci√≥n:
2024-10-23T10:35:07.275Z [[31merror[39m]: Error en la autenticaci√≥n:
2024-10-23T12:39:30.010Z [[31merror[39m]: Error creando registro: notNull Violation: Sede.localidad cannot be null,
notNull Violation: Sede.ip_asignada cannot be null
2024-10-23T12:39:30.020Z [[31merror[39m]: Error creando sede: notNull Violation: Sede.localidad cannot be null,
notNull Violation: Sede.ip_asignada cannot be null
2024-10-23T12:40:42.647Z [[31merror[39m]: Error creando registro: (conn:512776, no: 1054, SQLState: 42S22) Unknown column 'id_empresa' in 'field list'
sql: INSERT INTO `sedes` (`id_sede`,`id_empresa`,`nombre`,`direccion`,`localidad`,`provincia`,`pais`,`telefono`,`email`,`ip_asignada`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?); - parameters:['e8a13b22-5361-4c1c-9c8b-58f3fa7df05c','1a9bf1ea-ecea-...
2024-10-23T12:40:42.657Z [[31merror[39m]: Error creando sede: (conn:512776, no: 1054, SQLState: 42S22) Unknown column 'id_empresa' in 'field list'
sql: INSERT INTO `sedes` (`id_sede`,`id_empresa`,`nombre`,`direccion`,`localidad`,`provincia`,`pais`,`telefono`,`email`,`ip_asignada`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?); - parameters:['e8a13b22-5361-4c1c-9c8b-58f3fa7df05c','1a9bf1ea-ecea-...
2024-10-23T14:09:03.348Z [[31merror[39m]: Error obteniendo registros: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:09:03.349Z [[31merror[39m]: Error listando tipos de articulos: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:13:44.274Z [[31merror[39m]: Error obteniendo registros: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:13:44.274Z [[31merror[39m]: Error listando tipos de articulos: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:18:59.624Z [[31merror[39m]: Error obteniendo registros: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:18:59.624Z [[31merror[39m]: Error listando tipos de art√≠culos: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:20:00.117Z [[31merror[39m]: Error obteniendo registros: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:20:00.118Z [[31merror[39m]: Error listando tipos de art√≠culos: Cannot read properties of undefined (reading 'findAll')
2024-10-23T14:56:44.442Z [[31merror[39m]: Error creando registro: (conn:515978, no: 1364, SQLState: HY000) Field 'apellido' doesn't have a default value
sql: INSERT INTO `personas` (`id_persona`,`nombre`,`email`,`telefono`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?); - parameters:['f6971644-231f-4f67-9bb1-ba3879ef9100','Marina','msocolovsky@megatlon.com.ar','15-5705-2411','2024-10-23 14:56:44.418','2024-10-23...
2024-10-23T14:56:44.454Z [[31merror[39m]: Error creando persona: (conn:515978, no: 1364, SQLState: HY000) Field 'apellido' doesn't have a default value
sql: INSERT INTO `personas` (`id_persona`,`nombre`,`email`,`telefono`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?); - parameters:['f6971644-231f-4f67-9bb1-ba3879ef9100','Marina','msocolovsky@megatlon.com.ar','15-5705-2411','2024-10-23 14:56:44.418','2024-10-23...
2024-10-23T15:05:34.724Z [[31merror[39m]: Error al conectar la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1001ms
2024-10-23T15:13:41.883Z [[31merror[39m]: Error obteniendo sede: Persona is not defined
2024-10-23T15:14:44.076Z [[31merror[39m]: Error obteniendo registro por ID: Persona is not associated to Sede!
2024-10-23T15:14:44.076Z [[31merror[39m]: Error obteniendo sede: Persona is not associated to Sede!
2024-10-23T15:17:28.383Z [[31merror[39m]: Error obteniendo registro por ID: Rol is not associated to Persona!
2024-10-23T15:17:28.384Z [[31merror[39m]: Error obteniendo sede: Rol is not associated to Persona!
2024-10-23T15:19:30.939Z [[31merror[39m]: Error obteniendo sede: SedePersona is not defined
2024-10-23T15:19:47.879Z [[31merror[39m]: Error obteniendo registro por ID: sede_persona is not associated to Persona!
2024-10-23T15:19:47.880Z [[31merror[39m]: Error obteniendo sede: sede_persona is not associated to Persona!
2024-10-23T15:24:33.154Z [[31merror[39m]: Error creando registro: Cannot read properties of undefined (reading 'create')
2024-10-23T15:24:33.163Z [[31merror[39m]: Error creando asignaci√≥n: Cannot read properties of undefined (reading 'create')
2024-10-23T15:29:15.813Z [[31merror[39m]: Error obteniendo sede: Cannot read properties of undefined (reading '0')
2024-10-23T15:29:32.018Z [[31merror[39m]: Error obteniendo sede: Cannot read properties of undefined (reading '0')
2024-10-23T15:36:01.271Z [[31merror[39m]: Error obteniendo persona: Sede is not defined
2024-10-23T15:36:24.548Z [[31merror[39m]: Error obteniendo persona: SedePersona is not defined
2024-10-23T15:36:25.505Z [[31merror[39m]: Error obteniendo persona: SedePersona is not defined
2024-10-23T17:41:35.196Z [[31merror[39m]: Error creando registro: notNull Violation: TipoArticulo.nombre cannot be null
2024-10-23T17:41:35.204Z [[31merror[39m]: Error creando tipo de art√≠culo: notNull Violation: TipoArticulo.nombre cannot be null
2024-10-23T17:44:33.235Z [[31merror[39m]: Error creando registro: notNull Violation: Inventario.id_sede cannot be null
2024-10-23T17:44:33.244Z [[31merror[39m]: Error creando art√≠culo: notNull Violation: Inventario.id_sede cannot be null
2024-10-23T17:45:19.090Z [[31merror[39m]: Error creando registro: (conn:521607, no: 1364, SQLState: HY000) Field 'id_sede' doesn't have a default value
sql: INSERT INTO `inventarios` (`id_inventario`,`id_marca`,`modelo`,`id_tipo_articulo`,`service_tag`,`activo`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?,?); - parameters:['b9be5e7d-92b4-4595-a41e-35ade406d5e0','b1167afc-c8ea-4aa1-8e74-a37defd65c87','AEDQSE'...
2024-10-23T17:45:19.100Z [[31merror[39m]: Error creando art√≠culo: (conn:521607, no: 1364, SQLState: HY000) Field 'id_sede' doesn't have a default value
sql: INSERT INTO `inventarios` (`id_inventario`,`id_marca`,`modelo`,`id_tipo_articulo`,`service_tag`,`activo`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?,?); - parameters:['b9be5e7d-92b4-4595-a41e-35ade406d5e0','b1167afc-c8ea-4aa1-8e74-a37defd65c87','AEDQSE'...
2024-10-23T17:49:42.985Z [[31merror[39m]: Error registrando movimiento: (conn:521679, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?); - parameters:['ab4758a5-4331-4a0e-9485-8beec388ad39','972707e2-d977-4d65-a064-20f0ebf574d2','274f9d...
2024-10-23T17:49:43.066Z [[31merror[39m]: Error creando art√≠culo: (conn:521679, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?); - parameters:['ab4758a5-4331-4a0e-9485-8beec388ad39','972707e2-d977-4d65-a064-20f0ebf574d2','274f9d...
2024-10-23T17:53:08.767Z [[31merror[39m]: Error creando registro: notNull Violation: Inventario.id_sede cannot be null
2024-10-23T17:53:08.777Z [[31merror[39m]: Error creando art√≠culo: notNull Violation: Inventario.id_sede cannot be null
2024-10-23T17:55:25.212Z [[31merror[39m]: Error registrando movimiento: (conn:521835, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['a7b81c00-0dd1-4c9d-bcde-8f04be212491','d5f37d...
2024-10-23T17:55:25.272Z [[31merror[39m]: Error creando art√≠culo: (conn:521835, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['a7b81c00-0dd1-4c9d-bcde-8f04be212491','d5f37d...
2024-10-23T17:55:43.118Z [[31merror[39m]: Error registrando movimiento: (conn:521840, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['9abf4cea-fb83-4522-8a9c-392e18708ae5','e7c26c...
2024-10-23T17:55:43.165Z [[31merror[39m]: Error creando art√≠culo: (conn:521840, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['9abf4cea-fb83-4522-8a9c-392e18708ae5','e7c26c...
2024-10-23T17:56:52.593Z [[31merror[39m]: Error registrando movimiento: (conn:521899, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['1006545a-2984-4e5e-8028-4f4d526c27ad','dfa66f...
2024-10-23T17:56:52.644Z [[31merror[39m]: Error creando art√≠culo: (conn:521899, no: 1054, SQLState: 42S22) Unknown column 'id_sede' in 'field list'
sql: INSERT INTO `historico_inventario` (`id_historico`,`id_inventario`,`id_sede_origen`,`id_sede_destino`,`id_remito`,`fecha_movimiento`,`createdAt`,`updatedAt`,`id_sede`) VALUES (?,?,?,?,?,?,?,?,?); - parameters:['1006545a-2984-4e5e-8028-4f4d526c27ad','dfa66f...
2024-10-25T11:31:31.476Z [[31merror[39m]: Error al conectar la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1002ms
2024-10-25T18:01:48.759Z [[31merror[39m]: Error creando remito: (conn:645583, no: 1452, SQLState: 23000) Cannot add or update a child row: a foreign key constraint fails (`megasys2`.`remitos`, CONSTRAINT `remitos_ibfk_1` FOREIGN KEY (`id_sede`) REFERENCES `sedes` (`id_sede`) ON DELETE CASCADE ON UPDATE CASCADE)
sql: INSERT INTO `remitos` (`id_remito`,`id_sede`,`id_persona`,`id_user`,`fecha_remito`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?); - parameters:['1a9bf1ea-ecea-4765-a144-b289f4928d10','777eb9a5-2733-46cd-bdc9-421ac5bcfcc9','42fa6c79-8ae1-4a59-a0c9-1...
2024-10-25T18:01:48.760Z [[31merror[39m]: Error creando remito: (conn:645583, no: 1452, SQLState: 23000) Cannot add or update a child row: a foreign key constraint fails (`megasys2`.`remitos`, CONSTRAINT `remitos_ibfk_1` FOREIGN KEY (`id_sede`) REFERENCES `sedes` (`id_sede`) ON DELETE CASCADE ON UPDATE CASCADE)
sql: INSERT INTO `remitos` (`id_remito`,`id_sede`,`id_persona`,`id_user`,`fecha_remito`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?); - parameters:['1a9bf1ea-ecea-4765-a144-b289f4928d10','777eb9a5-2733-46cd-bdc9-421ac5bcfcc9','42fa6c79-8ae1-4a59-a0c9-1...
2024-10-25T18:03:11.307Z [[31merror[39m]: Error creando registro: notNull Violation: Sede.id_empresa cannot be null,
notNull Violation: Sede.nombre cannot be null,
notNull Violation: Sede.direccion cannot be null,
notNull Violation: Sede.localidad cannot be null,
notNull Violation: Sede.provincia cannot be null,
notNull Violation: Sede.pais cannot be null,
notNull Violation: Sede.telefono cannot be null,
notNull Violation: Sede.email cannot be null,
notNull Violation: Sede.ip_asignada cannot be null
2024-10-25T18:03:11.317Z [[31merror[39m]: Error creando sede: notNull Violation: Sede.id_empresa cannot be null,
notNull Violation: Sede.nombre cannot be null,
notNull Violation: Sede.direccion cannot be null,
notNull Violation: Sede.localidad cannot be null,
notNull Violation: Sede.provincia cannot be null,
notNull Violation: Sede.pais cannot be null,
notNull Violation: Sede.telefono cannot be null,
notNull Violation: Sede.email cannot be null,
notNull Violation: Sede.ip_asignada cannot be null
2024-10-25T18:04:01.757Z [[31merror[39m]: Error creando remito: articulos is not iterable
2024-10-25T18:04:01.757Z [[31merror[39m]: Error creando remito: articulos is not iterable
2024-10-25T18:05:07.648Z [[31merror[39m]: Error creando remito: (conn:645635, no: 1364, SQLState: HY000) Field 'id_remito_inventario' doesn't have a default value
sql: INSERT INTO `remito_inventario` (`id_remito`,`id_inventario`,`es_prestamo`,`fecha_devolucion`,`devuelto`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?); - parameters:[3,'78a90057-cc10-49ee-a555-121236f5ed3b',false,null,false,'2024-10-25 18:05:07.576','202...
2024-10-25T18:05:07.648Z [[31merror[39m]: Error creando remito: (conn:645635, no: 1364, SQLState: HY000) Field 'id_remito_inventario' doesn't have a default value
sql: INSERT INTO `remito_inventario` (`id_remito`,`id_inventario`,`es_prestamo`,`fecha_devolucion`,`devuelto`,`createdAt`,`updatedAt`) VALUES (?,?,?,?,?,?,?); - parameters:[3,'78a90057-cc10-49ee-a555-121236f5ed3b',false,null,false,'2024-10-25 18:05:07.576','202...
2024-10-25T18:08:10.114Z [[31merror[39m]: Error listando remitos: Sede is not defined
2024-10-25T18:08:10.114Z [[31merror[39m]: Error listando remitos: Sede is not defined
2024-10-25T18:09:04.133Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T18:09:04.134Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T18:11:10.174Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T18:11:10.174Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T18:11:47.848Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T18:11:47.849Z [[31merror[39m]: Error obteniendo remito: RemitoInventario is not associated to Remito!
2024-10-25T21:55:10.187Z [[31merror[39m]: Error creando remito: Transaction cannot be rolled back because it has been finished with state: commit
2024-10-25T21:59:05.040Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:67
    65|       <div>
    66|         <strong>Remito ID:</strong> <%= remito.id_remito %><br>
 >> 67|         <strong>Persona:</strong> <%= persona.nombre %> <br>
    68|         <strong>Sede:</strong> <%= sede.nombre %>
    69|       </div>
    70|       <div>

persona is not defined
2024-10-25T21:59:05.115Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T21:59:05.115Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:03:52.194Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:71
    69|       </div>
    70|       <div>
 >> 71|         <strong>Usuario:</strong> <%= user.nombre %>
    72|       </div>
    73|     </div>
    74| 

user is not defined
2024-10-25T22:03:52.337Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:03:52.337Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:05:38.932Z [[31merror[39m]: Error generando PDF: ENOENT: no such file or directory, open '/media/einar/Nuevo vol/Programacion/Megasys/remitos/remito_8.pdf'
2024-10-25T22:05:39.005Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:05:39.005Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:06:06.857Z [[31merror[39m]: Error generando PDF: ENOENT: no such file or directory, open '/media/einar/Nuevo vol/Programacion/Megasys/remitos/remito_9.pdf'
2024-10-25T22:06:06.902Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:06:06.902Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:06:31.592Z [[31merror[39m]: Error generando PDF: ENOENT: no such file or directory, open '/media/einar/Nuevo vol/Programacion/Megasys/remitos/remito_10.pdf'
2024-10-25T22:06:31.645Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:06:31.645Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:09:46.095Z [[31merror[39m]: Error registrando acci√≥n en hist√≥rico: notNull Violation: Historico.id_entidad cannot be null
2024-10-25T22:09:46.096Z [[31merror[39m]: Error creando remito: notNull Violation: Historico.id_entidad cannot be null
2024-10-25T22:18:46.834Z [[31merror[39m]: Error registrando acci√≥n en hist√≥rico: notNull Violation: Historico.id_entidad cannot be null
2024-10-25T22:18:46.834Z [[31merror[39m]: Error creando remito: notNull Violation: Historico.id_entidad cannot be null
2024-10-25T22:31:07.771Z [[31merror[39m]: Error generando PDF: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1001ms
2024-10-25T22:31:07.821Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T22:31:07.821Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:13:18.578Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:130
    128|             <tr>
    129|               <td><%= index + 1 %></td>
 >> 130|               <td><%= articulo.Inventario.modelo %></td>
    131|               <td><%= articulo.Inventario.service_tag || 'N/A' %></td>
    132|               <td><%= articulo.Inventario.num_serie || 'N/A' %></td>
    133|               <td><%= articulo.es_prestamo ? 'S√≠' : 'No' %></td>

Cannot read properties of undefined (reading 'modelo')
2024-10-25T23:13:18.619Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:13:18.619Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:17:47.098Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% articulos.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

articulos is not defined
2024-10-25T23:17:47.174Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:17:47.175Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:48:42.580Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:48:42.838Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:48:42.838Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:49:28.841Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:49:29.018Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:49:29.019Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:49:56.490Z [[31merror[39m]: Error al conectar la base de datos: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1000ms
2024-10-25T23:50:10.564Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:50:10.619Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:50:10.619Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:50:34.327Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:50:34.505Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:50:34.506Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:51:04.368Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:51:04.440Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:51:04.441Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:51:45.260Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% Inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.Inventario.modelo %></td>

Inventario is not defined
2024-10-25T23:51:45.349Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:51:45.349Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:52:29.984Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:130
    128|             <tr>
    129|               <td><%= index + 1 %></td>
 >> 130|               <td><%= articulo.Inventario.modelo %></td>
    131|               <td><%= articulo.Inventario.service_tag || 'N/A' %></td>
    132|               <td><%= articulo.Inventario.num_serie || 'N/A' %></td>
    133|               <td><%= articulo.es_prestamo ? 'S√≠' : 'No' %></td>

Cannot read properties of undefined (reading 'modelo')
2024-10-25T23:52:30.515Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:52:30.516Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:54:38.078Z [[31merror[39m]: Error creando remito: (conn:-1, no: 45012, SQLState: 08S01) Connection timeout: failed to create socket after 1001ms
2024-10-25T23:57:31.090Z [[31merror[39m]: Error generando PDF: /media/einar/Nuevo vol/Programacion/Megasys/megasys2/views/remitoTemplate.ejs:127
    125|         </thead>
    126|         <tbody>
 >> 127|           <% inventario.forEach((articulo, index) => { %>
    128|             <tr>
    129|               <td><%= index + 1 %></td>
    130|               <td><%= articulo.modelo %></td>

inventario is not defined
2024-10-25T23:57:31.818Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-25T23:57:31.818Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-26T00:11:52.363Z [[31merror[39m]: Error generando PDF: Cannot find module '../models/TipoArticulo'
Require stack:
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/services/RemitoService.js
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/controllers/remitoController.js
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/routes/remito.js
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/routes/routes.js
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/models/server.js
- /media/einar/Nuevo vol/Programacion/Megasys/megasys2/app.js
2024-10-26T00:11:52.413Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
2024-10-26T00:11:52.414Z [[31merror[39m]: Error creando remito: Error generando PDF del remito
